types/reactstrap/lib/Fade.d.ts

```diff
@@ -1,20 +1,28 @@
-import { CSSModule } from '../index';
+import { CSSModule } from "../index";
+
+export type EndHandler = (node: HTMLElement, done: () => void) => void;
+export type EnterHandler = (node: HTMLElement, isAppearing: boolean) => void;
+export type ExitHandler = (node: HTMLElement) => void;
 
 export interface FadeProps extends React.HTMLAttributes<HTMLElement> {
-  in?: boolean;
-  baseClass?: string;
-  baseClassIn?: string;
-  tag?: React.ReactType;
-  className?: string;
-  cssModule?: CSSModule;
-  transitionAppearTimeout?: number;
-  transitionEnterTimeout?: number;
-  transitionLeaveTimeout?: number;
-  transitionAppear?: boolean;
-  transitionEnter?: boolean;
-  transitionLeave?: boolean;
-  onLeave?: () => void;
-  onEnter?: () => void;
+    baseClass?: string;
+    baseClassActive?: string;
+    tag?: React.ReactType;
+    className?: string;
+    cssModule?: CSSModule;
+    // Props from 'react-transition-group/Transition'
```

Comment:
In addition to the earlier comment by @timc13, this should really import and extend react-transition-group/TransitionProps instead of manually specifying them here.

Reply:
Ok